aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AWSPinpointAnalyticsPlugin.java

```diff
@@ -320,7 +329,13 @@ public void unregisterGlobalProperties(@NonNull String... propertyNames) {
 
     @Override
     public void flushEvents() {
-        analyticsClient.submitEvents();
+        analyticsClient.submitEvents(analyticsEvents ->
+                Amplify.Hub.publish(HubChannel.ANALYTICS, HubEvent
+                    .create(AnalyticsHubEventNames.FLUSH_EVENTS, analyticsEvents)),
```

Comment:
Hub event class name should be `AnalyticsChannelEventName` for consistency with other hub event classes.